wrappers/python/tests/TestGromacsTopFile.py

```diff
@@ -151,6 +151,23 @@ def test_HydrogenMass(self):
         totalMass2 = sum([system2.getParticleMass(i) for i in range(system2.getNumParticles())]).value_in_unit(amu)
         self.assertAlmostEqual(totalMass1, totalMass2)
 
+    def test_VirtualParticle(self):
+        """Test virtual particle works correctly."""
+
+        top = GromacsTopFile('systems/bnz.top')
+        gro = GromacsGroFile('systems/bnz.gro')
+        system = top.createSystem()
+
+        self.assertEqual(26, system.getNumParticles())
+        self.assertEqual(1, len(top._moleculeTypes['BENX'].vsites2))
+
+        context = Context(system, VerletIntegrator(1*femtosecond),
+                          Platform.getPlatformByName('Reference'))
+        context.setPositions(gro.positions)
```

Comment:
I just realized this doesn't actually test the virtual site, because it's just using the positions loaded from the file.  After calling `setPositions()`, add the line

```python
context.computeVirtualSites()
```